### PR TITLE
NASA-PDS/devops#10

### PR DESCRIPTION
**Pull request by nutjob4life**
## 📜 Summary

Merge this and it just might make this repository look a bit more like the [pds-template-repo-python](https://github.com/NASA-PDS/pds-template-repo-python).

## 🩺 Test Data and/or Report

Before these changes:

```console
$ venv/bin/python setup.py test
…
███████Test█████████<unittest.runner.TextTestResult█num=15███████████failures=0>
```

After these changes:

```console
$ venv/bin/python setup.py test
…
███████Test█████████<unittest.runner.TextTestResult█num=15███████████failures=0>
```

No change = good 😇

## 🧩 Related Issues

- NASA-PDS/devops#10

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Note: this project is not set up for unit, functional, integration,
```
Comment (MJJoyce) [inline tox.ini:1]: Is everything in this file supposed to be commented out?

**Comment by MJJoyce** [comment]
Looks good to me!

File: tox.ini (hunk 1 above)
Comment (nutjob4life) [reply tox.ini:1]: Yes. See the comment at the top for the reason why.

Comment (MJJoyce) [reply tox.ini:1]: Guess it helps if I read. Sorry sir!

Comment (nutjob4life) [reply tox.ini:1]: @MJJoyce it's cool ☺️

**Review by MJJoyce — APPROVED** [review]

**Pull request merged**